styleguide/_themes/global/scss/cta.scss

```diff
@@ -0,0 +1,42 @@
+.cta {
+  text-align: center;
+}
+
+.cta-headerWrapper {
+  @include make-md-column(12);
+}
+
+.cta-button {
+  @include button;
+  @include button-lead;
+  display: table;
+}
+
+.cta-buttonWrapper {
+  @include make-md-column(12);
+  margin-bottom: 3px;
+}
+
+.cta-iconWrapper {
+  @include make-md-column(12);
+}
+
+.cta-row {
+  @include make-row;
+}
+
+.cta-subheadWrapper {
+  @include make-md-column(12);
+
+  .cta-subhead {
```

Comment:
Is this nested for greater specificity to get the CSS properties to take effect?